### ISSUE TITLE
License Violation

### ISSUE THREAD

**Issue by arj3090** [devdjung/Excel_EIP#1]
This project is a violation to the AdvancedHMI software license. In order to comply, you must release the full source of your project and it must be part of the original AdvancedHMI solution. The AdvancedHMI license does not allow parts of the solution to be separated.

**Comment by devdjung** [comment]
If you want the sources, just ask me, and you get it.

2016-02-08 13:25 GMT+01:00 arj3090 notifications@github.com:

**Comment by arj3090** [comment]
Yes I am interested in seeing how you made the connection from AdvancedHMI to Excel.

**Comment by devdjung** [comment]
at this moment i'm in holiday, until 24.02, so i send you the sources on
25.02.

Best regards.

2016-02-09 14:30 GMT+01:00 arj3090 notifications@github.com:

**Comment by devdjung** [comment]
now, the source is available to download.
